prover/prover_dal/migrations/20240524123522_add-patch-columns-for-semver.down.sql

```diff
@@ -30,3 +30,42 @@ ALTER TABLE gpu_prover_queue_fri_archive
 
 ALTER TABLE prover_fri_protocol_versions
     DROP IF EXISTS protocol_version_patch;
```

Comment:
It should be DROP COLUMN IF EXISTS if I'm not mistaken. Please test migration with `cd prover/prover_dal && DATABASE_URL=postgres://postgres:notsecurepassword@localhost/prover_local cargo sqlx migrate run`, `DATABASE_URL=postgres://postgres:notsecurepassword@localhost/prover_local cargo sqlx migrate revert`, `DATABASE_URL=postgres://postgres:notsecurepassword@localhost/prover_local cargo sqlx migrate run` to check that it works

Reply:
Sorry, without COLUMN is also works, down.sql almost works, you should only remove
```
ALTER TABLE prover_fri_protocol_versions
     DROP CONSTRAINT prover_fri_protocol_versions_pkey CASCADE;
```
for there because it gets remove when patch column is dropped